frontend/public/components/operator-lifecycle-manager/create-operand.tsx

```diff
@@ -6,6 +6,7 @@ import { safeDump } from 'js-yaml';
 import * as _ from 'lodash-es';
 import { PropertyPath } from 'lodash';
 import * as classNames from 'classnames';
+import { Switch } from 'patternfly-react';
```

Comment:
Is there a reason to prefer importing from `patternfly-react`, rather than `@patternfly/react-core`(already imported in this file)?

Reply:
👍 Thanks for the explanation